Documentation/distributions.md

```diff
@@ -95,10 +95,16 @@ sudo emerge rkt
 
 ## NixOS
 
-rkt can be installed on NixOS using the following command:
+On NixOS enable rkt by adding the following line in `/etc/nixos/configuration.nix`:
 
 ```
-nix-env -iA rkt
+virtualisation.rkt.enable = true;
```

Comment:
Correct me if I'm wrong, but it looks like this accomplishes the following three things:

1. Makes the rkt binary available for that configuration
1. Enables the rkt metadata services
1. Enables the rkt gc timer

I think there are people who may want 1 but not 2 or 3 (especially given the metadata service is only useful in a a subset of usecases).

Perhaps those differences should be cited so a nixos user may opt to instead just install the pkg without the services if they want?